pkg/cmd/repo/sync/git.go

```diff
@@ -0,0 +1,98 @@
+package sync
+
+import (
+	"os/exec"
+
+	"github.com/cli/cli/git"
+	"github.com/cli/cli/internal/run"
+)
+
+//counterfeiter:generate . gitClient
```

Comment:
I see only 2 of the `gitClient` methods explicitly stubbed in tests, and yet the tests pass and I'm sure that they invoke more than just 2 methods. Does counterfeiter have default return values for methods that were not explicitly stubbed? I find this to be a dealbreaker because I would prefer that tests have to explicitly set up every stub for every invocation of the method.

Reply:
Yes 100%. That is what is happening here and it totally threw me off. I agree, I dislike this behavior and would rather everything blew up if the function is called and is not stubbed.